sklearn/utils/fixes.py

```diff
@@ -314,3 +316,22 @@ def in1d(ar1, ar2, assume_unique=False, invert=False):
     from ._scipy_sparse_lsqr_backport import lsqr as sparse_lsqr
 else:
     from scipy.sparse.linalg import lsqr as sparse_lsqr
+
+
+if sys.version_info < (2, 7, 0):
+    # partial cannot be pickled in Python 2.6
+    # http://bugs.python.org/issue1398
+    class partial(object):
```

Comment:
Did you implement that yourself? This looks like the kind of thing where one forgets the edge cases ;)